### PR TITLE
fix: substrate/dev dep, update changelog

### PR DESCRIPTION
**Pull request by TarikGul**
Update @substrate/dev dep, 

Update the changelog from an emergency release last week.

### CODE REVIEW

**Review by niklasad1 — COMMENTED** [review]
just fix the date in the CHANGELOG and it's good to go

**Review by jsdw — APPROVED** [review]

**Comment by jsdw** [comment]
our of interest, what's the difference between substrate-exec-* and *?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [9.1.2](https://github.com/paritytech/substrate-api-sidecar/compare/v9.1.1..v9.1.2) (2021-08-02)
```
[the remark this answers is not in the document]
Comment (TarikGul) [reply CHANGELOG.md:5]: This is actually correct 9.1.2 was released last week. Just forgot to update the changelog because it was an emergency release

**Review by niklasad1 — APPROVED** [review]

**Comment by TarikGul** [comment]
@jsdw so we run all binaries via scripts from `@substrate/dev` which allows us to have all of our deps and configs in a centralized package, and under our control is all. It's more for consistency

**Pull request merged**